daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1230,10 +1251,12 @@ void IEEE1588Port::getDeviceTime
 
 void IEEE1588Port::becomeMaster( bool annc ) {
 	port_state = PTP_MASTER;
-	// Start announce receipt timeout timer
-	// Start sync receipt timeout timer
+	// Stop announce receipt timeout timer
 	clock->deleteEventTimer( this, ANNOUNCE_RECEIPT_TIMEOUT_EXPIRES );
```

Comment:
This doesn't need attention as part of this pull request, but these changes made me wonder if we ought to update the management of the announce receipt timer to follow the same pattern the others do now, with helper functions and locks.